src/library/Uno.Toolkit.Material/Styles/Controls/v2/BottomTabBar.xaml

```diff
@@ -0,0 +1,707 @@
+	                ResourceKey="OnPrimaryContainerFocusedBrush" />
+	<StaticResource x:Key="M3MaterialBottomFabTabBarItemPressedStateOverlay"
+	                ResourceKey="OnPrimaryContainerPressedBrush" />
+	<StaticResource x:Key="M3MaterialBottomFabTabBarItemDisabledStateOverlay"
+	                ResourceKey="OnSurfaceDisabledLowBrush" />
+
+	<x:Double x:Key="M3MaterialBottomTabBarFontSize">12</x:Double>
+	<FontWeight x:Key="M3MaterialBottomTabBarFontWeight">Medium</FontWeight>
+	<x:Double x:Key="M3MaterialBottomTabBarHeight">80</x:Double>
+	<GridLength x:Key="M3MaterialBottomTabBarGridLengthHeight">80</GridLength>
+	<x:Double x:Key="FabItemVerticalOffset">-32</x:Double>
+	<x:Double x:Key="M3MaterialBottomTabBarItemIconHeight">16</x:Double>
+	<x:Double x:Key="M3MaterialBottomTabBarItemIconWidth">16</x:Double>
+	<Thickness x:Key="M3MaterialBottomTabBarItemContentMargin">0,0,0,6</Thickness>
+	<Thickness x:Key="M3MaterialBottomTabBarItemContentOnlyMargin">12,0</Thickness>
+	<CornerRadius x:Key="M3MaterialBottomTabBarItemActiveIndicatorCornerRadius">16</CornerRadius>
+	<Thickness x:Key="M3MaterialBottomTabBarItemActiveIndicatorPadding">26,10</Thickness>
+	<!--  Code can be removed when this issue is fixed in Uno:  -->
+	<!--  https://github.com/unoplatform/uno/issues/7393  -->
+	<!--  AjustedMaterialBottomTabBarHeight = MaterialBottomTabBarHeight + Half size of the Fab TabBarItem (including ElevatedView margin)  -->
+	<x:Double x:Key="AjustedMaterialBottomTabBarHeight">108</x:Double>
+	<Thickness x:Key="M3MaterialBottomFabTabBarItemPadding">16</Thickness>
+	<x:Double x:Key="M3MaterialBottomFabTabBarItemContentWidthOrHeight">18</x:Double>
+	<CornerRadius x:Key="M3MaterialBottomFabTabBarItemCornerRadius">16</CornerRadius>
+	<x:Double x:Key="M3MaterialBottomFabTabBarItemIconTextPadding">12</x:Double>
+
+
+	<!--  Material Bottom TabBar  -->
+	<mobile:Style x:Key="M3MaterialBottomTabBarStyle"
+	              TargetType="utu:TabBar">
+		<Setter Property="Background"
+		        Value="{ThemeResource M3MaterialBottomTabBarBackground}" />
+		<Setter Property="IsTabStop"
+		        Value="False" />
+		<Setter Property="toolkit:VisibleBoundsPadding.PaddingMask"
+		        Value="Bottom" />
+		<Setter Property="ItemsPanel">
+			<Setter.Value>
+				<ItemsPanelTemplate>
+					<utu:TabBarListPanel />
+				</ItemsPanelTemplate>
+			</Setter.Value>
+		</Setter>
+		<Setter Property="ItemContainerStyle"
+		        Value="{StaticResource M3MaterialBottomTabBarItemStyle}" />
+		<!--  Workaround until this issue is fixed, can be removed after  -->
+		<!--  https://github.com/unoplatform/uno/issues/7393  -->
+		<Setter Property="VerticalContentAlignment"
+		        Value="Bottom" />
+		<Setter Property="Template">
+			<Setter.Value>
+				<ControlTemplate TargetType="utu:TabBar">
+					<!--  Code can be uncommented when this issue is fixed in Uno:  -->
+					<!--  https://github.com/unoplatform/uno/issues/7393  -->
+					<!--<Grid x:Name="TabBarGrid"
```

Comment:
i would suggest using `mc:Ignorable` with `todo:` for commenting out xaml stuff
better formatting and less likely to mess up when comment/uncommenting from parent node